builder/lxd/communicator.go

```diff
@@ -12,6 +12,7 @@ import (
 	"os/exec"
 	"path/filepath"
 	"syscall"
+	"strings"
```

Comment:
Since the imports aren't in alphabetical order here, our linters fail.

Could you run goimports on the file and update this PR? I couldn't do it since I don't have the permission to push on your remote.

Reply:
Should be better now

Reply:
All good indeed, many thanks @Novakov !